system-addon/content-src/components/Search/_Search.scss

```diff
@@ -13,15 +13,12 @@
   cursor: default;
   display: flex;
   height: $search-height;
-  // The extra 1px is to account for the box-shadow being outside of the element
-  // instead of inside. It needs to be like that to not overlap the inner background
-  // color of the hover state of the submit button.
```

Comment:
I tried to reproduce this code comment but I couldnt see a difference? So I removed the horizontal margin so the dropdown is the right width.

Reply:
Hmm.. @k88hudson added that comment in #3785.. Doesn't look like either 1px need to be around anymore?